ansible/roles/tower-settings-update/tasks/main.yml

```diff
@@ -3,12 +3,12 @@
 
 - name: Set the value of AWX_PROOT_BASE_PATH
   tower_settings:
-    name:               "{{ tower_setting.key }}"
-    value:              "{{ tower_setting.value }}"
-    tower_host:         "{{ tower_hostname }}"
-    tower_username:     "{{ tower_admin_username | default('admin') }}"
-    tower_password:     "{{ tower_admin_password }}"
-    tower_verify_ssl:   "{{ tower_verify_ssl | default(false) }}"
```

Comment:
We should allow this indent style, need to review yamllint settings to permit this.